### ISSUE TITLE
Sync crashes because of duplicate seasons

### ISSUE THREAD

**Issue by mregni** [mregni/EmbyStat#950]
Looks like we also have an issue with duplicate season index numbers while syncing. Needs the same fix as #949 probably.

```2019-09-26 03:24:46.0300 [INFO] THETVDB-CLIENT Call to THETVDB: [url=https://api.thetvdb.com/series/72514/episodes?page=12 ]https://api.thetvdb.com/series/72514/episodes?page=12 [/url]
2019-09-26 03:24:46.1380 [ERROR] MEDIASYNC-JOB Can't seem to process show Looney Tunes, check the logs for more details! 
2019-09-26 03:24:46.1380 [ERROR] System.InvalidOperationException: Sequence contains more than one matching element
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at EmbyStat.Jobs.Jobs.Sync.MediaSyncJob.ProgressMissingEpisodesAsync(Show show, CancellationToken cancellationToken) in d:\a\1\s\EmbyStat.Jobs\Jobs\Sync\MediaSyncJob.cs:line 307
   at EmbyStat.Jobs.Jobs.Sync.MediaSyncJob.GetMissingEpisodesFromTvdbAsync(IReadOnlyList`1 shows, CancellationToken cancellationToken) in d:\a\1\s\EmbyStat.Jobs\Jobs\Sync\MediaSyncJob.cs:line 278 System.InvalidOperationException: Sequence contains more than one matching element
   at System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)
   at EmbyStat.Jobs.Jobs.Sync.MediaSyncJob.ProgressMissingEpisodesAsync(Show show, CancellationToken cancellationToken) in d:\a\1\s\EmbyStat.Jobs\Jobs\Sync\MediaSyncJob.cs:line 307
   at EmbyStat.Jobs.Jobs.Sync.MediaSyncJob.GetMissingEpisodesFromTvdbAsync(IReadOnlyList`1 shows, CancellationToken cancellationToken) in d:\a\1\s\EmbyStat.Jobs\Jobs\Sync\MediaSyncJob.cs:line 278```

**Comment by Happy2Play** [comment]
What do you mean by duplicate seasons?

**Comment by Happy2Play** [comment]
I found my issue in the Series folder, some how I had two additional seasons folders without spaces in there names and Emby was only showing one of them.

example
Z:\ServerFolders\Videos\Kids-Animation\TV Shows\Looney Tunes (1929)\Season 1966
Z:\ServerFolders\Videos\Kids-Animation\TV Shows\Looney Tunes (1929)\Season1966

Removing the problem folders, and ensuring the problem folders were removed from Emby's database resolves the issue.

**Comment by mregni** [comment]
Yeah, by duplicate season folders I mean 2 folders with the same index number. In my inital implementation I do a SingleOrDefault() select because I presumed only 1 season for each indexNumber could exist. Seems when you have indeed some naming errors like you mention Emby can handle it by creating 2 seasons with the same indexNumber.

So I updated my code so it can handle them as well. Maybe it's a good idea to implement some overview somewhere that shows when you have this kind of duplicate season folders.
